### ISSUE TITLE
How can I fix NaN loss in VAE autoencoder?

### ISSUE THREAD

**Issue by JasOlean** [keras-team/keras#12371]
latent_dim = 20
learning_rate = 0.0001
train_set = (10240, 42)
test_set = (8192, 42)
nb_epch = 100
autoencoder.fit(x_train, x_train,
                    epochs=nb_epoch,
                    batch_size=batch_size,
                    shuffle=True,
                    validation_data=(x_test, x_test))

When I train simple autoencoder, it is fine. But when I train Variational Autoencoder, I got nan loss error like this:

8192/8192 [==============================] - 0s 39us/step - loss: nan - acc: 0.1409 - val_loss: nan - val_acc: 0.1499
Epoch 64/100
8192/8192 [==============================] - 0s 39us/step - loss: nan - acc: 0.1409 - val_loss: nan - val_acc: 0.1499
Epoch 65/100
8192/8192 [==============================] - 0s 39us/step - loss: nan - acc: 0.1409 - val_loss: nan - val_acc: 0.1499
Epoch 66/100
8192/8192 [==============================] - 0s 39us/step - loss: nan - acc: 0.1409 - val_loss: nan - val_acc: 0.1499
Epoch 67/100
8192/8192 [==============================] - 0s 39us/step - loss: nan - acc: 0.1409 - val_loss: nan - val_acc: 0.1499

How can I fix this? Please help me!!

**Comment by bayethiernodiop** [comment]
hello check if you don't have nan values in your inputs
